roles/edpm_ovn_bgp_agent/templates/ovn_bgp_agent.yaml.j2

```diff
@@ -0,0 +1,24 @@
+start_order: 0
+image: "{{ edpm_ovn_bgp_agent_image }}"
+net: host
+pid: host
+cgroupns: host
+restart: always
+privileged: true
+healthcheck:
+  test: '/openstack/healthcheck'
+volumes:
+  {% set edpm_ovn_bgp_agent_volumes = [] %}
+  {%- set edpm_ovn_bgp_agent_volumes =
+          edpm_ovn_bgp_agent_volumes +
+          edpm_ovn_bgp_agent_common_volumes %}
+  {% if edpm_ovn_bgp_agent_internal_tls_enable|bool %}
+  {%- set edpm_ovn_bgp_agent_volumes =
+         edpm_ovn_bgp_agent_volumes +
+         edpm_ovn_bgp_agent_common_volumes +
+         edpm_ovn_bgp_agent_tls_volumes -%}
+  {% endif %}
+  {{ edpm_ovn_bgp_agent_volumes }}
```

Comment:
just a quick question. does this really render the value in the expected format ?

Reply:
I copied from how it was done in nova side nova_compute.yaml.j2

It works without tls, but I have not tested it with tls enabled

Reply:
ok then let's leave this implementation now and revisit this later.